app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        self.__class__.people[name] = self
+
+    def __repr__(self) -> str:
+        return f"Person(name={self.name}, age={self.age})"
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    person_dict = {}
+
+    for person_data in people:
+        name = person_data["name"]
+        age = person_data["age"]
+        person = Person(name, age)
+        person_list.append(person)
+        person_dict[name] = person
```

Comment:
why did you create this dict ?

Reply:
To use the get() method. I couldn't pass one test without it.